smallfile_plugin.py

```diff
@@ -12,34 +12,47 @@
 import csv
 from arcaflow_plugin_sdk import plugin
 
-from smallfile_schema import WorkloadParams, WorkloadResults, WorkloadError, smallfile_schema, output_params_schema, output_results_schema, output_rsptimes_schema
+from smallfile_schema import (
```

Comment:
In the future, such large code reformats should be made in their own commit to help reviewers be sure they care understanding the nature of the changes for a given PR.  This is a trade-off that individuals can make, but the discipline can really help improve the code and speed up the process of reviews and merges.